src/CMakeLists.txt

```diff
@@ -1009,9 +1030,9 @@ install(TARGETS osdmaptool DESTINATION bin)
 
 set(ceph_psim_srcs
   tools/psim.cc)
-add_executable(ceph-psim ${ceph_psim_srcs})
-target_link_libraries(ceph-psim global)
-install(TARGETS ceph-psim DESTINATION bin)
+add_executable(ceph_psim ${ceph_psim_srcs})
```

Comment:
@alimaredia i reconsidered this change a little bit: actually there are a bunch of executables named as "ceph_" in ceph-test package, see ceph.spec.in.  probably we can change them all at once, if you think we should go this way. and please note this is a user visible change. i'd like to put them into another PR, so it's more visible to other reviewers. what do you think?